vsphere/vsphere.go

```diff
@@ -83,7 +88,13 @@ func NewClient(cfg *VSphereConfig) (cloudops.Ops, error) {
 	logrus.Debugf("  Datacenter: %s", vmObj.Datacenter.Name())
 	logrus.Debugf("  VMUUID: %s", cfg.VMUUID)
 
-	configMap, err := configmap.New(vSphereDataStoreLock, nil, 0, 420, 5*time.Second, 10*time.Second)
+	storeInstance, err := store.GetStoreWithParams(
+		storeParams.Kv,
+		storeParams.SchedulerType,
+		storeParams.InternalKvdb,
+		vSphereDataStoreLock,
+		420*time.Second,
+		150*time.Second)
```

Comment:
Should this be 180*time.Second instead to make it same as the default kvdb lock timeout of 3 minutes?

Reply:
I'd probably reduce that to 100 seconds. The lock will kinda serialize node brinup, so I'd reduce it futher.